hydropandas/io/bro.py

```diff
@@ -422,8 +423,9 @@ def get_full_metadata_from_gmw(bro_id, tube_nr):
     return meta
 
 
-def get_tube_nrs_from_gmw(bro_id):
-    """returns all tube numbers from a groundwater monitoring well (gmw)
+@lru_cache()
+def _get_gmw_from_bro_id(bro_id, retries=0):
```

Comment:
is retries number of retries or a counter that goes up to 5?

Reply:
I see it's recursive now. So nvm, but maybe good to allow max_retries to be configurable?

Reply:
Good idea, I've added rcParams for this